general-concepts/manifest/text.xml

```diff
@@ -15,10 +15,9 @@ by the package. This is used to verify integrity upon fetching them.
 </p>
 
 <p>
-To generate the <c>Manifest</c>, use <c>ebuild foo.ebuild manifest</c>,
-<c>repoman manifest</c>, or <c>pkgdev manifest -m</c>. You may want to set
-<c>GENTOO_MIRRORS=</c> while calling it to fetch distfiles from their original
-locations immediately.
+To generate the <c>Manifest</c>, use <c>ebuild foo.ebuild manifest</c> or
+<c>pkgdev manifest -m</c>. You may want to set <c>GENTOO_MIRRORS=</c> while
+calling it to fetch distfiles from their original locations immediately.
```

Comment:
Encouraging `<c>pkgdev manifest -m</c>` and then saying `You may want to set <c>GENTOO_MIRRORS=</c>` conflicts with each other :)

I've never had problems with pure `pkgdev manifest`... even think it prefers GENTOO_MIRRORS.

Reply:
I agree. I don't understand that either, but it's not my objective to resolve that in this patch.

Looks like @thesamesam added this in commit 00c1824be4d4bd2a72a54be9846846478d25ada1. Let's resolve that in a follow up.

Reply:
GENTOO_MIRRORS usage is disabled for `pkgdev manifest` by default, passing the -m option enables them. See https://github.com/pkgcore/pkgdev/blob/main/src/pkgdev/scripts/pkgdev_manifest.py#L37 ... or in other words read the man page. :smile: